### ISSUE TITLE
Clouddriver fails to start after upgrading to Spinnaker 1.29.0

### ISSUE THREAD

**Issue by ashleykleynhans** [spinnaker/spinnaker#6733]
### Issue Summary:

After upgrading Spinnaker from 1.28.1 to 1.29.0, Clouddriver fails to start.

### Cloud Provider(s):

- AWS
- ECS
- Google

### Environment:

- Ubuntu 22.04 LTS VM
- Halyard 1.51.0
- Spinnaker 1.29.0

### Feature Area:

Clouddriver

### Description:

I've upgraded Spinnaker successfully several times, all the way up until version 1.28.1.
However, after upgrading to Spinnaker version 1.29.0, Clouddriver fails to start with the following error:

```
***************************
APPLICATION FAILED TO START
***************************

Description:

Field credentialsRepository in com.netflix.spinnaker.clouddriver.google.deploy.validators.TerminateAndDecrementGoogleServerGroupDescriptionValidator required a bean of type 'com.netflix.spinnaker.credentials.CredentialsRepository' that could not be found.

The injection point has the following annotations:
	- @org.springframework.beans.factory.annotation.Autowired(required=true)


Action:

Consider defining a bean of type 'com.netflix.spinnaker.credentials.CredentialsRepository' in your configuration.
```

### Steps to Reproduce:

1. Install Halyard v1.51.0 on a Ubuntu 22.04 LTS VM.
2. Use Halyard to install Spinnaker 1.28.1 with AWS, ECS and Google as enabled cloud providers.
3. `hal config version edit --version 1.29.0`
4. `sudo hal deploy apply`

### Additional Details:

Since the error message implies that this issue is related to the Google provider, I disabled the Google provider and then Clouddriver starts successfully, so this seems to be some sort of issue related specifically to the Google provider.

**Comment by kirangodishala** [comment]
I am looking into it

**Comment by jasonmcintosh** [comment]
I THINK this is resolved at this point?  Note Google Appengine had the same issue... I fixed that recently...

**Comment by ashleykleynhans** [comment]
This is fixed in 1.29.1.
